SingularityUI/app/handlebarsHelpers.coffee

```diff
@@ -103,3 +103,6 @@ Handlebars.registerHelper 'humanizeFileSize', (bytes) ->
 Handlebars.registerHelper 'usernameFromEmail', (email) ->
     return '' if not email
     email.split('@')[0]
+
+Handlebars.logger.level = 0;
+
```

Comment:
@tpetr the Handlebars logger level makes `{{log ...}}` values actually show in console (equivalent of console.log)- we don't do this in our code but not sure if we want to leave this setting on or off. In Rails, there's also a logger level and it's on in production as well.